aiohttp/websocket.py

```diff
@@ -30,6 +30,15 @@
 
 Message = collections.namedtuple('Message', ['tp', 'data', 'extra'])
 
+UNPACK_HEADER = Struct('!BB').unpack
```

Comment:
You can optimize it more by not using struct for trivial cases at all. Single byte is a such.